nautobot/core/views/mixins.py

```diff
@@ -231,6 +231,7 @@ class NautobotViewSetMixin(GenericViewSet, AccessMixin, GetReturnURLMixin, FormV
     serializer_class = None
     table_class = None
     notes_form_class = NoteForm
+    custom_action_permission_map = None
```

Comment:
Unused?

Reply:
Yep thought I got rid of it. Will remove pending other comments to make sure this is the route to go. :-)